File: informatique/tps/tp_b2/repete.ml
let () =
  if Array.length Sys.argv = 3 then
    let n = int_of_string Sys.argv.(1) and s = Sys.argv.(2) in
    for i = 1 to n do
      print_string s;
      print_char ' ';
    done
  else
    Printf.printf "usage : %s nb str répète nb fois la chaîne de caractères" Sys.argv.(0)

File: informatique/tps/tp_b2/repete.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char* argv[]) {
	if (argc == 3) {
		int n = atoi(argv[1]);
		char* s = argv[2];

		for(int i = 0; i < n; ++i) {
			printf("%s ", s);
		}
		printf("\n");
	} else {
		printf("usage : %s nb str repete nb fois la chaîne de caractères", argv[0]);
	}
}
